devbox.json

```diff
@@ -39,6 +39,11 @@
       "test-projects-only": "DEVBOX_RUN_PROJECT_TESTS=1 go test -v -timeout ${DEVBOX_GOLANG_TEST_TIMEOUT:-30m} ./... -run \"TestExamples|TestScriptsWithProjects\"",
       "update-examples":    "devbox run build && go run testscripts/testrunner/updater/main.go",
       "tidy":               "go mod tidy",
+
+      "test-linux-amd64": [
```

Comment:
it feels a bit weird to have this in the top level devbox.json, maybe we add a devbox.json to testscripts? It could call out to this devbox in order to build the linux devbox

not a blocker